### ISSUE TITLE
false positive, use functional writing style

### ISSUE THREAD

**Issue by larshp** [larshp/abaplint#117]
```abap
          CALL METHOD (method_name)
            PARAMETER-TABLE parameters.
```

**Comment by larshp** [comment]
after #125
